pkg/coremodel/datasource/lineage.cue

```diff
@@ -0,0 +1,38 @@
+package datasource
+
+import "github.com/grafana/thema"
+
+thema.#Lineage
+name: "datasource"
+seqs: [
+    {
+        schemas: [
+            { // 0.0
+                name: string
+                type: string
+                access: *"proxy" | "direct"
+                url?: string
+                password: string
+                user: string
+                database?: string
+                // no longer used, now in secureJsonData
+//                basicAuth: bool
+//                basicAuthUser: string
+//                basicAuthPassword: string
+                withCredentials: bool | *false
+                isDefault: bool | *false
+                jsonData?: [string]: _
```

Comment:
curiousity: jsonData is `[string]: _` vs. secureJsonData is `[string]: bytes` - why?